wp-content/themes/core/components/image/css/image.pcss

```diff
@@ -22,14 +13,15 @@
 .c-image__link {}
 
 /* -----------------------------------------------------------------------------
- * Image Component: Image Element
+ * Image Component: Wrapper element
  *
- * An `img` for foreground usages and a `div` for background usages.
+ * Generally, a `figure` for foreground usages and a `div` for background usages.
+ * Customizable via component's attributes.
  * ----------------------------------------------------------------------------- */
 
```

Comment:
Same note on all these `&.c-image--` classes. I think we can reduce the specificity on them and just put them at the top level below `.c-image`, @dpellenwood?

Reply:
Agreed. Please un-nest these selectors.